lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobinSpec.hs

```diff
@@ -588,7 +650,7 @@ prop_performSelection_small minCoinValueFor costFor (Blind (Small criteria)) =
     checkCoverage $
     cover 30 (balanceSufficient criteria)
         "balance sufficient" $
-    cover 30 (not $ balanceSufficient criteria)
+    cover 25 (not $ balanceSufficient criteria)
```

Comment:
These coverage values were reduced because it seems my generator changes have reduced the occurrence of these paths.

I"m not sure how to fix this.

Reply:
It's not necessarily a big problem. I'd be concerned if we had to reduce coverage to a very low percentage, but `25` seems fine here. 👍🏻